frontend/test/metabase/scenarios/dashboard-filters/dashboard-filters-source.cy.spec.js

```diff
@@ -94,8 +94,8 @@ describe("scenarios > dashboard > filters", () => {
 const setupStructuredQuestionSource = () => {
   cy.findByText("Values from a model or question").click();
   modal().within(() => {
-    cy.findByText("Saved Questions").click();
-    cy.findByText("GUI categories").click();
+    cy.findByPlaceholderText(/Search for a question/).type("Categories");
```

Comment:
Should we have another case that could return results not found?
e.g.
<img width="545" alt="image" src="https://user-images.githubusercontent.com/1937582/210374319-3b2000a1-6a39-4092-b094-333a886db485.png">

Reply:
I believe this should be tested on the picker level and not by us. This looks like an issue with the underlying component.

Reply:
Sounds good. Do we have tests for this for DataPicker component, if not should we create new ones?

Reply:
@WiNloSt currently @kulyk is working on adding tests for the picker